README.md

```diff
@@ -3,6 +3,9 @@
 # Aeternity Elixir Full Node
 This is an elixir full node implementation of the aeternity specification.
 
+Compatibility to the erlang aeternity implementation is documented in [docs/aeternity-erlang-compatibility.md](docs/aeternity-erlang-compatibility.md).
```

Comment:
I think this should be written more like:

Tests for checking compatibility to the epoch aeternity implementation are documented in...

Reply:
this should be more of a documentation with future versions just stating changes, without more specific cases. The sample cases are just for outsiders to quickly get up and running to check these the documented cases.